apps/meteor/client/views/admin/users/AdminUsersPage.tsx

```diff
@@ -104,13 +120,33 @@ const AdminUsersPage = (): ReactElement => {
 					</TabsItem>
 				</Tabs>
 				<PageContent>
+					{isSeatsCapExceeded && (
```

Comment:
use the same logic used here `SubscriptionCalloutLimits`, also check with pms if there is a real need to introduce a new texts to explain the same thing